package.json

```diff
@@ -16,12 +18,17 @@
   },
   "main": "index.js",
   "devDependencies": {
+    "babel-core": "^6.24.1",
+    "babel-preset-env": "^1.4.0",
     "chai": "^3.5.0",
+    "eslint": "^3.19.0",
+    "eslint-plugin-json": "^1.2.0",
     "grunt": "^1.0.1",
```

Comment:
Yes! I found my package.json was not getting updated even after the npm uninstall command, that the reason they are still there ... no problem ... will update that too